scss/tech.scss

```diff
@@ -16,9 +16,15 @@
     left: -10px;
     position: absolute;
     top: -10px;
+    transition: left 0.5s ease;
+    transition: top 0.5s ease;
```

Comment:
you can add multiple transition like this `transition: color .2s, text-shadow .2s;`, even you can make a variable fior the `0.5s ease` as it is repeated